examples/test_examples.py

```diff
@@ -19,18 +19,26 @@
 # limitations under the License.
 
 
+from unittest import skip
+
+from neo4j.v1 import basic_auth, TRUST_ON_FIRST_USE, TRUST_SIGNED_CERTIFICATES
 from test.util import ServerTestCase
 
+# Do not change the contents of this tagged section without good reason*
 # tag::minimal-example-import[]
-from neo4j.v1 import GraphDatabase, basic_auth
```

Comment:
Shouldn't this include `basic_auth` though? For 99% of first-time users, this import line is what they should use.